addons/mail/views/mail_menus.xml

```diff
@@ -9,16 +9,23 @@
     />
     <menuitem
         id="mail.main_menu_discuss"
-        name="Conversations"
+        name="Discuss"
         parent="mail.menu_root_discuss"
         action="action_discuss"
         sequence="1"
     />
+    <menuitem
+        id="mail.menu_channel"
```

Comment:
This is dangerous, we need to be really careful.

If we were to use `mail.menu_channel` in code, it would crash on already existing DB in this version.

(Just for adding the item in new db, it is probably fine).